master/buildbot/test/unit/test_db_logs.py

```diff
@@ -465,8 +467,48 @@ def thd(conn):
             'content': lz4.dumps(line),
             'compressed': 3})
 
-    # TODO: test compressing with >64k
-    # TODO: test compressing compressed size >64k
+    @defer.inlineCallbacks
+    def do_addLogLines_huge_log(self, NUM_CHUNKS=3000, chunk=u'xy' * 70 + '\n' * 3):
```

Comment:
This creates a string with three newlines at the end. Was that the intention?

    >>> u'xy' * 70 + '\n' * 3
    u'xyxyxyxyxyxyxyxyxyxyxyxyxyxyxyxyxyxyxyxyxyxyxyxyxyxyxyxyxyxyxyxyxyxyxyxyxyxyxyxyxyxyxyxyxyxyxyxyxyxyxyxyxyxyxyxyxyxyxyxyxyxyxyxyxyxyxyxyxyxy\n\n\n'